src/main/java/org/folio/rest/impl/CheckinReceivePiecesHelper.java

```diff
@@ -542,7 +540,7 @@ private void updateOrderStatus(List<PoLine> poLines) {
   private CompletableFuture<ReceiptStatus> calculatePoLineReceiptStatus(int expectedPiecesQuantity, PoLine poLine,
       List<Piece> pieces) {
     // Fully Received:If receiving and there is no expected piece remaining
-    if (!isCheckin(poLine) && expectedPiecesQuantity == 0) {
+    if (expectedPiecesQuantity == 0) {
```

Comment:
Instead of just removing method call, it should be replaced by `!poLine.getCheckinItems()`. Current version introduces regression...

Reply:
Done